discover/reference/articles/knowledge-base-configuration.markdown

```diff
@@ -0,0 +1,683 @@
+menu. However, some of the configuration settings are set to variables and must 
+be overwritten through either a portal properties file or a configuration file. 
+For example, the Knowledge Base Article configuration has the value shown below 
+for the *Social bookmarks types* setting:
+
+    ${server-property://com.liferay.portal/social.bookmark.types}
+    
+the `server-property` prefix specifies that this value is set through a server
+property. You can view the default value by going to *Configuration* &rarr; 
+*Server Administration* and selecting the *Properties* tab and 
+*Portal Properties* sub-tab. From here you can Search for the property you 
+want(e.g., `social.bookmark.types`) to see the default value. To overwrite the 
+property's default value, add the updated property values to the 
+`portal-ext.properties` file in your app server's root directory.
+
+The second kind of variable prefix is `resource`. For example, the
+*Email KBArticle suggestion in progress body* setting for the Knowledge Base 
+Service configuration has the following value:
+
+    ${resource:com/liferay/knowledge/base/dependencies/email_kb_suggestion_in_progress_body.tmpl}
+
+This points to the resource that provides the template. To overwrite these 
+resources, you need to either change the configuration in the System Settings UI 
+to point to your new file, or using a configuration file in your app server's 
+`osgi/config` directory, you need to overwrite the property to point to the 
+absolute file path of the new file. 
+
++$$$
+
+**Note:** Once you've exported the configuration files, as explained in the 
+[System Settings](/discover/portal/-/knowledge_base/7-0/system-settings#exporting-and-importing-configurations) 
+User Guide documentation, the configuration files are located in a folder called
+`liferay-system-settings`. Within this folder you'll find the 
+configuration(.cfg) files, named after the full class name for that file. For
+example, the *Knowledge Base Service* configuration file is called
+`com.liferay.knowledge.base.configuration.KBGroupServiceConfiguration.cfg` and
+the *Knowledge Base Section* configuration file is called
+`com.liferay.knowledge.base.web.configuration.KBSectionPortletInstanceConfiguration.cfg`.
+To overwrite these files, you must use the same name for your configuration file, 
+and place it in the `osgi/config` directory.
+
+$$$
+
+You can view the current configuration files and settings by exporting the 
+configuration file, as explained in the [System Settings](/discover/portal/-/knowledge_base/7-0/system-settings#exporting-and-importing-configurations) 
+User Guide documentation.
+    
+### Equivalent Configuration Properties for 6.2 Portal Properties [](id=equivalent-configuration-properties-for-6-2-portal-properties)
+
+Here is a list of the properties previously found in Knowledge Base's 
+`portlet.properties` and the new properties to use in the Knowledge Base 
+configuration files.
```

Comment:
A lot of these are the email settings, which you should overwrite with a config file(so the original is preserved), in which case you would need to use these properties, but yeah I see what you are saying. I can add a note to the beginning that encourages using the system settings when possible. I'll try to add a third column, but I think it may be too wide for the article. I'll give it a try though.